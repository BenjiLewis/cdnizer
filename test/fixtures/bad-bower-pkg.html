<script src="js/bad-pkg/bad-pkg.js"></script>
