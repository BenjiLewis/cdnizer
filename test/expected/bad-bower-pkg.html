<script src="//cdn/bad-pkg/1.0.0/bad-pkg.js"></script>
